app/lib/status_updater.rb

```diff
@@ -141,6 +170,25 @@ def patient_statuses_to_import
         end
   end
 
+  def patient_session_registration_statuses_to_import
+    @patient_session_registration_statuses_to_import ||=
+      patient_sessions
+        .eager_load(:patient)
+        .includes(session: %i[session_dates])
+        .find_each
+        .flat_map do |patient_session|
+          unless programme_ids_per_year_group.key?(
```

Comment:
```suggestion
year_group = patient_session.patient.year_group
        
next [] unless programme_ids_per_year_group.key?(year_group)
```

Reply:
The code has changed a bit since this comment, it should be more readable now.